argo-cd.yaml

```diff
@@ -39,7 +39,7 @@ pipeline:
       unset GOFLAGS
 
       # CVE-2023-2253
```

Comment:
should any CVEs be added to these comments?

Reply:
this PR address move to stable release from beta , all the files have either  # Handle GHSA-hqxw-f8mx-cpmw or # CVE-2023-2253 as part of the comments